tests/python/contrib/test_arm_compute_lib/test_runtime.py

```diff
@@ -0,0 +1,98 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Arm Compute Library runtime tests."""
+
+import numpy as np
+
+import tvm
+from tvm import relay
+
+from .infrastructure import skip_runtime_test, build_and_run, verify
+from .infrastructure import Device
+
+
+def test_multiple_ops():
+    """
+    Test multiple operators destined for ACL.
+    ACL will expect these ops as in 2 separate functions.
```

Comment:
It seems to me that this test doesn't really test the number of ACL functions directly. For example, if ACL runtime supports multiple kernel ops in the future, then this test will still pass even if the partitioning result is unexpected. I would suggest adding tests for `partition_for_arm_compute_lib` to check if the transformed Relay graph matches our expectation, including composite functions and partitioned ACL functions.

Reply:
You're correct, I'll add this to all the tests as ideally this should be checked each time.